math/math_util.h

```diff
@@ -111,6 +136,42 @@ T Ramp(const T x, const T x_min, const T x_max, const T y_min, const T y_max) {
   return (y_min + (x - x_min) / (x_max - x_min) * (y_max - y_min));
 }
 
+ template <typename T>
+ T ProbabilityDensityGuassian(const T& sample, const T& mean, const T& stddev) {
```

Comment:
These functions do not belong in `geometry.h`, but in `math_util.h`

Reply:
Or maybe a new file for statistics?

Reply:
I have pulled these into `statistics.h`.